src/synthesizer/python/mod.rs

```diff
@@ -0,0 +1,645 @@
+use crate::code::{CodeBuffer, IndentOptions};
+use crate::ir::conditions::ConditionIr;
+use crate::ir::importer::ImportInstruction;
+use crate::ir::mappings::MappingInstruction;
+use crate::ir::outputs::OutputInstruction;
+use crate::ir::reference::{Origin, PseudoParameter, Reference};
+use crate::ir::resources::{ResourceInstruction, ResourceIr};
+use crate::ir::CloudformationProgramIr;
+use crate::parser::lookup_table::MappingInnerValue;
+use indexmap::IndexMap;
+use std::borrow::Cow;
+use std::io;
+use std::rc::Rc;
+use voca_rs::case::camel_case;
+
+use super::Synthesizer;
+
+const INDENT: Cow<'static, str> = Cow::Borrowed("  ");
+
+pub struct Python {
+    // TODO: Put options in here for different outputs in Python
+}
+
+impl Python {
+    #[cfg_attr(coverage_nightly, no_coverage)]
+    #[deprecated(note = "Prefer using the Synthesizer API instead")]
```

Comment:
Since we don't consider this module stable yet, we should probably just delete old functionality rather than deprecating, especially since all of this functionality is brand new. Why add an immediately deprecated function?

Reply:
Yeah just another thing I did to keep parity with the ts synthesizer. I went ahead and deleted it